.github/workflows/ci-complete.yml

```diff
@@ -118,3 +118,17 @@ jobs:
           description: 'The build scan was successfully published'
           context: 'Gradle Build Scan / Java ${{ matrix.java }}'
           state: 'success'
+
+  update-cached-tag:
+    # Skip this workflow if the CI run was skipped or cancelled
+    if: (github.event.workflow_run.conclusion == 'success' || github.event.workflow_run.conclusion == 'failure')
```

Comment:
In this case, we only want to run if the workflow was successful.

Reply:
Oh, thanks! I will fix it!